### PR TITLE
remove log_mad*

### PR DESCRIPTION
**Pull request by pindge**
```
The config issue is that the three style_tmad_*dev  styles should be deleted (keeping the style_tmad_*dev_std  styles).
```

### CODE REVIEW

File: dev/services/wms/ows_refactored/tmad/ows_tmad_cfg.py [hunk 1]
```diff
@@ -5,109 +5,13 @@
...
-}
 
 style_tmad_sdev_std = {
-    "name": "arcsec_sdev",
+    "name": "log_sdev",
     "title": "SMAD",
     "abstract": "Good for cropland and forest",
     "index_function": {
-        "function": "datacube_ows.band_utils.single_band_arcsec",
+        "function": "datacube_ows.band_utils.single_band_offset_log",
```
Comment (SpacemanPaul) [inline dev/services/wms/ows_refactored/tmad/ows_tmad_cfg.py:14]: No, we want to keep arcsec as the function - which I guess means we have to change the name too.

**Review by SpacemanPaul — CHANGES_REQUESTED** [review]
We want to keep the arcsec function for sdev, sorry.

File: dev/services/wms/ows_refactored/tmad/ows_tmad_cfg.py (hunk 1 above)
Comment (pindge) [reply dev/services/wms/ows_refactored/tmad/ows_tmad_cfg.py:14]: so 3 `arcsec` styles?

Comment (pindge) [reply dev/services/wms/ows_refactored/tmad/ows_tmad_cfg.py:14]: because they are missing

Comment (SpacemanPaul) [reply dev/services/wms/ows_refactored/tmad/ows_tmad_cfg.py:14]: No bcdev and edev use log  - sdev uses arcsec.

I know it looks inconsistent, but it's how I tuned it with Dale.

Comment (SpacemanPaul) [reply dev/services/wms/ows_refactored/tmad/ows_tmad_cfg.py:14]: If you want it to look consistent, use sdev_scaled, edev_scaled, etc.

**Review by SpacemanPaul — APPROVED** [review]
Looks right to me.

**Pull request merged**